src/components/FormPartial/FormPartial.tsx

```diff
@@ -8,14 +8,21 @@ const FormPartial: FormPartialType<any> = (
     const options = fieldOptions[field] as FieldOptionsValueType<string>;
     const error = validationErrors && validationErrors[field];
     const { component } = options;
-    const componentOptions = {
+
+    let commonComponentOptions = {
```

Comment:
Since this is referenced but not changed, maybe it could be a const?

```suggestion
    const commonComponentOptions = {
```